src/site/content/en/fast/debugging-layout-shifts/index.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: post
+title: Debugging layout shifts
+authors:
+  - katiehempenius
+date: 2021-03-11
+hero: image/j2RDdG43oidUy6AL6LovThjeX9c2/N65l8ccOUEDESpnnEkje.png
+alt: Picture of a layout shift.
+description: |
+  Learn how to identify and fix layout shifts.
```

Comment:
Not sure why, but this description is not showing up on the blog page https://deploy-preview-4870--web-dev-staging.netlify.app/blog/